tikv-client/src/main/java/com/pingcap/tikv/codec/MyDecimal.java

```diff
@@ -51,18 +69,26 @@
   private static final int[] dig2bytes = new int[] {0, 1, 1, 2, 2, 3, 3, 4, 4, 4};
 
   // The following are fields of MyDecimal
-  private int digitsInt;
-  private int digitsFrac;
-  private boolean negative;
-  private int[] wordBuf = new int[maxWordBufLen];
-
+  private int _digitsInt;
+  private int _digitsFrac;
+  private boolean _negative;
+  private int[] _wordBuf = new int[maxWordBufLen];
```

Comment:
Point of adding initial underscore?

Reply:
It is added to avoid confusion since we use similar name as local variable in toStirng method.

Reply:
Who have bigger scope? Local variables or class members?
And who has the higher priority when hitting naming conflict? The bigger scope of the small one?

Reply:
class memory has larger scope. I got your point. I will change it later.